dev/com.ibm.ws.logging_2_fat/publish/files/server_test_log_chatty.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+    Copyright (c) 2025 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="Server for testing traceFileName=test.log (chatty)">
+    <include location="../fatTestPorts.xml"/>
+    <featureManager>
+        <feature>timedexit-1.0</feature>
+    </featureManager>
+    <logging
+        traceSpecification="*=info:com.ibm.ws.kernel.*=all:com.ibm.ws.config.*=all"
+        traceFileName="test.log"
+        rolloverStartTime="00:00"
```

Comment:
Don't need to include rollover* logging config, since its already configured in the server.env.